modules/graalvm/22.2.0-java11/configure

```diff
@@ -5,8 +5,7 @@ SOURCES_DIR=/tmp/artifacts
 
 ls -l ${SOURCES_DIR}
 tar xzf ${SOURCES_DIR}/${FILENAME} -C /opt
-ls -l /opt
-mv /opt/graalvm-ce-${GRAALVM_VERSION}* /opt/graalvm
+mv /opt/graalvm-ce-${GRAALVM_VERSION} /opt/graalvm
```

Comment:
@gastaldi if this didn't cause any issues I would prefer to keep it as it was.

Reply:
Done